src/firebase.js

```diff
@@ -5,26 +5,14 @@ import * as config from './config';
 
 
 let messaging = null;
-let supported = false;
 
 export const notificationApiIsSupported = () =>
 	'Notification' in window &&
 	'serviceWorker' in navigator &&
 	'PushManager' in window
```

Comment:
Can this result ever change (during a single page load)? I.e., could this be just a `const` boolean instead of a function?